docs/react-native-install.md

```diff
@@ -1,13 +1,44 @@
-# React Native Installation Guide for Windows
+# React Native RICHO THETA Demo Installation Guide for Windows
+
+![react-windows-header](images/react_native/windows/reactwindows.svg)
 
 ## Overview
 
-Guide covers the installation process of React Native on Windows OS. Another section will cover the installation for Mac OS.
+Guide covers the installation process of React Native on **Windows OS** so that we can run the R. Another section will cover the installation for Mac OS. For this installation you will need to install **node.js** if you haven't already. Their will be videos and links in the resource section for the installation of node.js on your computer. Article also covers how to build the `theta-client` and make it available to `demo-react-native`. As well as building the `demo-react-native` and running it on an android emulator. My results will be shown in the article with the **THETA X** running the react native demo app provided by the RICOH API.
```

Comment:
likely mean React Native demo app.  

Press line return (enter key) at end of each line to make review easier